ballerina/build.gradle

```diff
@@ -85,7 +85,7 @@ task clean {
 }
 
 task build {
-    build.dependsOn ':qm-ballerina:qm:build'
+    build.dependsOn ':qm-ballerina:api_inspectionlot_srv:build'
```

Comment:
This should depend on updateDocumentationFiles task too

Reply:
Updated in [Update docs.json,settings.gradle and build.gradle for PR build](https://github.com/ballerina-platform/module-ballerinax-sap.s4hana.qm/pull/11/commits/e0fb7feb53a487803c5141f8f87b17a0f2cfce26) please check